_javascript/pwa/sw.js

```diff
@@ -5,7 +5,18 @@ importScripts(`${baseurl}/assets/js/data/swconf.js`);
 const purge = swconf.purge;
 
 function verifyUrl(url) {
-  const requestPath = new URL(url).pathname;
+  const requestUrl = new URL(url);
+  const requestPath = requestUrl.pathname;
+
+  if (!requestUrl.protocol.startsWith('http')) {
```

Comment:
```js
if (!requestUrl.protocol.startsWith('http')) {
  return false;
}
```

This check is to exclude requests like `chrome-extension://kiodaajmphnkcajieajajinghpejdjai/js/inject.js` to prevent errors in the console.

Reply:
This makes sense.